lib/src/rules/prefer_inlined_adds.dart

```diff
@@ -17,13 +17,13 @@ Declare elements in list literals inline, rather than using `add` and
 **BAD:**
 ```dart
 var l = ['a']..add('b')..add('c');
-var l2 = ['a']..addAll(['b', 'c'])
+var l2 = ['a']..addAll(['b', 'c']);
 ```
 
 **GOOD:**
 ```dart
 var l = ['a', 'b', 'c'];
-var 2 = ['a', 'b', 'c'];
+var 12 = ['a', 'b', 'c'];
```

Comment:
Should this still be `l2` to match the bad example?

```suggestion
var l2 = ['a', 'b', 'c'];
```

Reply:
Ha.  Absolutely.

Reply:
Thanks!